astring.cpp

```diff
@@ -196,6 +196,12 @@ char *AString::Str()
 	return str;
 }
 
+
+const char *AString::const_str() const
+{
+	return str;
+}
+
```

Comment:
We needed a way to get a const char* back out in some cases, so I added this to AString even though my intent is to get rid of it entirely at some point.